Makefile

```diff
@@ -294,12 +294,14 @@ endif
 .PHONY: update-deps
 update-deps:
 	go mod tidy
+	cd scripts && go mod tidy
 
 .PHONY: verify-deps
 verify-deps:
 	go mod verify
 	go mod tidy
 	@git diff --exit-code -- go.mod go.sum
+	cd scripts && go mod verify && go mod tidy && git diff --exit-code -- go.mod go.sum
```

Comment:
I feel like creating a new `verify-tools-deps` rule invoking `verify-deps` from the scripts directory would be a bit cleaner no?

Reply:
Whops, this change was supposed to be on the other MR!